packages/office-addin-lint/src/lint.ts

```diff
@@ -36,8 +47,14 @@ export function getLintFixCommand(files: string): string {
 }
 
 export function performLintFix(files: string) {
-  const command = getLintFixCommand(files);
-  execCommand(command);
+  try{
+    const command = getLintFixCommand(files);
+    execCommand(command);
+    usageDataHelper.sendUsageDataSuccessEvent("fix");
```

Comment:
`performLintFix` is a better name. It might be good pattern to use the function name, unless there is a good reason not to do so. I like consistency like that.